sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -849,6 +904,23 @@ public partial class ActionGroupPatch
         public bool? IsEnabled { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ActionType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ActionType>
```

Comment:
The name is too generic, so please use `rename-mapping` to rename it with a more describe name

Reply:
Renamed to "MonitorWorkspaceActionType"